qrest/resource.py

```diff
@@ -208,6 +208,11 @@ class Resource(ABC):
 
     response: Response
 
+    @abstractmethod
+    def clone(self):
+        """Return a new instance created with the same parameters as the current one."""
+        ...
```

Comment:
I'm not familiar with a `...` syntax in Python. Is this a placeholder, or did I just learn something new?

Reply:
You learned something new, just as I did a few months ago ;) It's called the Ellipsis literal and can be used as a kind-of no-operation but it has some other uses too. For an overview, have a look at [this article](https://florian-dahlitz.de/articles/what-is-pythons-ellipsis-object).

In hindsight, just using `pass` would have been clearer. The linked article says it best:
> Yes, it can be used as a drop-in replacement for the pass keyword, but you probably should not do it, because there is a reason why everyone uses pass. Using it in articles to indicate further code lines should be fine, though!

I saw it being used in a presentation on Python type-hinting. In that presentation it was used to denote an "irrelevant snippet of code".